react-three/src/styles.css

```
@@ -29,6 +29,35 @@ body {
+  animation: turtleSwim 4.5s ease-in-out infinite;
+  pointer-events: auto;
+  will-change: transform;
+}
+
+.turtle-canvas {
+  display: block;
+  width: 100% !important;
+  height: 100% !important;
+}
+
+@keyframes turtleSwim {
```

Stylelint (17.14.0)

[error] Expected keyframe name "turtleSwim" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)